src/esp/core/esp.h

```diff
@@ -106,6 +137,14 @@ inline std::ostream& operator<<(std::ostream& os, const box3f& bbox) {
   return os << "{min:" << bbox.min() << ",max:" << bbox.max() << "}";
 }
 
+inline Corrade::Utility::Debug& operator<<(Corrade::Utility::Debug& out,
+                                           const box3f& bbox) {
+  using Corrade::Utility::Debug;
+  return out << "{min:" << Debug::nospace << bbox.min() << Debug::nospace
+             << ",max:" << Debug::nospace << bbox.max() << Debug::nospace
+             << "}";
+}
```

Comment:
Oh, Eigen has AABBs? Interesting, missed those.

Continuing with the suggestion from above -- as of mosra/magnum-integration@789f8132842536f077548bc4f2d2c3685e9f491e, converting to a `Range3D` in order to print it with Debug should work as well :)